src/Calculator/SettingsPage.xaml

```diff
@@ -0,0 +1,164 @@
+<Page x:Class="CalculatorApp.SettingsPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:Windows10version1809="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 7)"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:CalculatorApp"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      Background="{ThemeResource AppChromeAcrylicHostBackdropMediumLowBrush}"
+      d:Height="465"
+      d:Width="445"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <Style x:Key="CategoryNameTextBlockStyle" TargetType="TextBlock">
+            <Setter Property="FontSize" Value="20"/>
+            <Setter Property="FontWeight" Value="SemiBold"/>
+            <Setter Property="OpticalMarginAlignment" Value="TrimSideBearings"/>
+        </Style>
+    </Page.Resources>
+
+    <ScrollViewer>
+        <Grid x:Name="PageGrid" AutomationProperties.LandmarkType="Main">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="1*"
+                           MinHeight="{StaticResource HamburgerHeight}"
```

Comment:
This seems strange to me. If I'm understanding correctly, then you are trying to explicitly give space to the NavView hamburger button. Are the other pages needing to do this?

Reply:
Hi @sanderl, I'm doing that because this is not the NavView button, this is the back button.